src/schemas/Array.js

```diff
@@ -24,6 +24,12 @@ export const normalize = (schema, input, parent, key, visit, addEntity, visitedE
 export const denormalize = (schema, input, unvisit) => {
   schema = validateSchema(schema);
   let found = true;
+  if (input === undefined && schema) {
```

Comment:
the case of the missing array. recurse to discover missing entities (for instance in objects).